docs/products/kafka/howto/viewing-resetting-offset.rst

```diff
@@ -0,0 +1,145 @@
+-----------------------------------------------------------------
+
+The ``kafka-consumer-groups.sh`` tool enables to manage consumer group offsets, the following commands are available.
+
+List the active consumer groups
+'''''''''''''''''''''''''''''''''''''''''
+
+To list the currently active consumer groups use the following command replacing the ``demo-kafka.my-project.aivencloud.com:17072`` with the Aiven for Apache Kafka service URI:
+
+::
+
+    kafka-consumer-groups.sh \
+        --bootstrap-server demo-kafka.my-project.aivencloud.com:17072 \
+        --command-config consumer.properties \
+        --list
+
+An example of the ``list`` option, showing two consumer groups, is the following:
+
+.. code:: text
+
+    my-group
+    my-group-2
+
+Retrieve the details of a consumer group
+''''''''''''''''''''''''''''''''''''''''''''''''
+
+To retrieve the details of a consumer group use the following command replacing the ``demo-kafka.my-project.aivencloud.com:17072`` with the Aiven for Apache Kafka service URI and the ``my-group`` with the required consumer group name:
+
+::
+
+    kafka-consumer-groups.sh \
+        --bootstrap-server demo-kafka.my-project.aivencloud.com:17072 \
+        --command-config consumer.properties \
+        --group my-group \
+        --describe
+          
+An example of the ``describe`` option, showing the details of the ``my-group`` consumer groups, is the following:
+
+.. code:: text
+
+    GROUP           TOPIC           PARTITION  CURRENT-OFFSET  LOG-END-OFFSET  LAG             CONSUMER-ID                HOST            CLIENT-ID
+    my-group        test-topic      0          5509            5515            6               rdkafka-39404560-f8f2-4b0b /151.62.82.140  rdkafka
+
+List the current members of a consumer group
+''''''''''''''''''''''''''''''''''''''''''''
+
+To retrieve the current members of a consumer group use the following command replacing the ``demo-kafka.my-project.aivencloud.com:17072`` with the Aiven for Apache Kafka service URI and the ``my-group`` with the required consumer group name:
+
+::
+
+    kafka-consumer-groups.sh \
+        --bootstrap-server demo-kafka.my-project.aivencloud.com:17072 \
+        --command-config consumer.properties \
+        --group my-group \
+        --describe \
+        --members
+
+An example of the ``--members`` option, showing the members of the ``my-group`` consumer groups, is the following:
+
+.. code:: text
+
+    GROUP           CONSUMER-ID                                  HOST            CLIENT-ID       #PARTITIONS
+    my-group        rdkafka-a4c0a09c-8c6e-457e-bf9e-354a8e2f4bb8 /151.62.82.140  rdkafka         0
+    my-group        rdkafka-39404560-f8f2-4b0b-9518-811e2eb20074 /151.62.82.140  rdkafka         1
+
+
+Reset the offset of a consumer group
+'''''''''''''''''''''''''''''''''''''
+
+Resetting the consumer group offset might be needed when the topic parsing needs to start at a specific (non default) offset.
```

Comment:
I'd simplify this as well to avoid passive voice - "Resetting the consumer group offset might be needed" - "You might want to reset  the consumer group offset when the topic parsing needs to start at a specific (non default) offset"